### ISSUE TITLE
Food recall warning: Issue accessing the content under "Additional information" with screen reader

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#128]
URL: https://hcdev.openplus.ca/en/alert-recall/test-certain-happy-veggie-world-hearts-choices-and-vegefarm-brand-vegan-and-0
User Agent: Chrome (v94), Firefox (v92)
Language: English and French
Tool: JAWS 2020

Issue Description:
There are some issues with the content provided below “Additional information”. When testing with JAWS, it doesn’t identify the label of each tab or when activating the tab nothing happened or state of the tabs is announced. Content associated with the selected tab panel is not read. Item “Media and public inquires” in the list is marked up differently than other items. It is a button with expand/collapse feature and work with screen reader as expected.

Further, when testing NVDA 2021.1, there are issues as well like It doesn't announce how many tabs are there, which is the current tab, state of the activated tab.

![image](https://user-images.githubusercontent.com/12256067/137554184-13ddb57a-cb39-4bbb-bf66-11fee0c13092.png)

Steps to reproduce:
1.	Open the URL provided above
2.	Launch JAWS / NVDA screen reader
3.	Navigate the page using the "Tab" key

Expected results: The tab interface content should work correctly with the screen readers. Probably, it would be better that all the items should be marked up the same way as the “Media and public inquires” as visually other items don’t look like they are tab interface content rather expand/collapse content.

**Comment by echouin** [comment]
Fixed.

**Comment by mmiron-HC** [comment]
fixed. No longer tabs
